internal/contracts/manager.go

```diff
@@ -58,7 +58,10 @@ type Manager interface {
 	AddContractAPIListener(ctx context.Context, ns, apiName, eventPath string, listener *core.ContractListener) (output *core.ContractListener, err error)
 	GetContractListenerByNameOrID(ctx context.Context, ns, nameOrID string) (*core.ContractListener, error)
 	GetContractListeners(ctx context.Context, ns string, filter database.AndFilter) ([]*core.ContractListener, *database.FilterResult, error)
+	GetContractListenerByIDGlobal(ctx context.Context, nameOrID string) (*core.ContractListener, error)
+	GetContractListenersGlobal(ctx context.Context, filter database.AndFilter) ([]*core.ContractListener, *database.FilterResult, error)
```

Comment:
Global collections are going away very soon (everything needs to be retrieved within a namespace)

Reply:
This is going (only the cross-namespace operations query will be retained at the end of this PR - which FFTM relies upon for the very short term)